android/app/proguard-rules.pro

```diff
@@ -0,0 +1,31 @@
+# Add project specific ProGuard rules here.
+# You can control the set of applied configuration files using the
+# proguardFiles setting in build.gradle.kts.kts.
+#
+# For more details, see
+#   http://developer.android.com/guide/developing/tools/proguard.html
+
+-dontobfuscate
+
+# Required for the patcher to function correctly
+-keep class app.revanced.patcher.** {
```

Comment:
Why is this necessary?

Reply:
I copied it from compose manager
but yes, there are no reason to disable obfuscation

Reply:
This should be clarified first. @Axelen123 Whats the reason for this?